test/index.spec.js

```diff
@@ -352,14 +352,16 @@ describe('Index', function() {
     });
 
     it('should send if there is something in the queue', function() {
-      var queue = [{name: 'walterwhite'}, {name: 'jessepinkman'}];
+        var queue = [{name: 'walterwhite', metadata: 'heisenberg'}, {name: 'jessepinkman'}];
```

Comment:
looks like you have too many spaces at the start of this line. It should be two spaces per indentation.